docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -17,16 +17,18 @@ The Alerts page offers various ways for you to organize and triage detection ale
 [role="screenshot"]
 image::images/view-alert-details.png[View details button, 200]
 
+* View the rule that created an alert. Click the name in the *Rule* column to open the rule's details page.
+
 * Filter for a specific rule in the KQL bar (for example, `kibana.alert.rule.name :"SSH (Secure Shell) from the Internet"`). KQL autocomplete is available for `.alerts-security.alerts-*` indices.
 
 * Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.
 
 * Visualize and group alerts by specific fields in the visualization section. Use the buttons on the left to select a view type (*Summary*, *Trend*, *Counts*, or *Treemap*), and use the menus on the right to select the ECS fields used for grouping alerts. Refer to <<visualize-alerts>> for more on each view type.
 
-* Hover over a value in the data grid to display available inline actions, such as *Filter In*, *Filter Out*, and *Add to timeline investigation*. Click the expand button to open a full context menu of options, including *Show top values*, *Copy to Clipboard*, and *View rule details*. The available options vary based on the type of data.
+* Hover over a value to display available <<inline-actions, inline actions>>, such as *Filter In*, *Filter Out*, and *Add to timeline*. Click the expand icon for more options, including *Show top _x_* and *Copy to Clipboard*. The available options vary based on the type of data.
 +
 [role="screenshot"]
-image::images/inline-actions-menu.gif[width=75%][height=75%][Animation of using the inline additional actions menu]
+image::images/inline-actions-menu.png[Inline additional actions menu,45%]
```

Comment:
This screenshot seems kind of small to me, maybe try a bigger % like 60 or 70?

Reply:
Let's try 55% -- even 60 looks kinda overwhelming on my screen
```suggestion
image::images/inline-actions-menu.png[Inline additional actions menu,55%]
```